gleam.toml

```diff
@@ -10,7 +10,7 @@ links = [
 ]
 
 [dependencies]
-gleam_stdlib = "~> 0.18"
+gleam_stdlib = "~> 0.23"
```

Comment:
Could you revert this please? This would prevent older stdlib users from using future `gleam_http` releases.

Reply:
Done! And I've ignored `manifest.toml` as well.